src/common/s3_adapter.cpp

```diff
@@ -132,6 +132,8 @@ void InitS3AdaptorOptionExceptS3InfoOption(Configuration* conf,
         &s3Opt->bpsReadMB));
     LOG_IF(FATAL, !conf->GetUInt64Value("s3.throttle.bpsWriteMB",
         &s3Opt->bpsWriteMB));
+    LOG_IF(FATAL, !conf->GetBoolValue("s3.useVirtualAddressing",
```

Comment:
how about func InitS3AdaptorOption? need add this new config?

Reply:
InitS3AdaptorOption will call InitS3AdaptorOptionExceptS3InfoOption